packages/prettier-config/lib/index.js

```diff
@@ -1,4 +1,13 @@
-module.exports = {
+/** @typedef {import('prettier').Options} PrettierOptions */
+
+/**
+ * @typedef WPPrettierOptions
```

Comment:
It feels like something we could recommend in docs - prefixing with WP when extending an existing 3rd part type. I think we use it also for React types.

Well, I found an exception:
https://github.com/WordPress/gutenberg/blob/bee1bf91958c71c733be4514b42cfa136243b3eb/packages/element/src/react.js#L34

Still, it feels okay.

Reply:
Yeah, I don't know that we need to be so strict here. There are guidelines in the documentation about the "WP" prefix, which I think could be sufficient.

>A `WP` prefix should be applied to all custom types.

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#custom-types